test/test_t2hahn.py

```diff
@@ -160,12 +160,12 @@ def test_experiment_config(self):
         exp = T2Hahn(0, [1, 2, 3, 4, 5])
         loaded_exp = T2Hahn.from_config(exp.config())
         self.assertNotEqual(exp, loaded_exp)
-        self.assertTrue(self.experiments_equiv(exp, loaded_exp))
+        self.assertTrue(self.json_equiv(exp, loaded_exp))
```

Comment:
What's the reason for this change?

Reply:
The self.experiment_equiv() wasn't working anymore in the test so I looked into Ramsey Experiment code and saw that it was updated with those lines.